src/structures/APIMessage.js

```diff
@@ -92,7 +92,7 @@ class APIMessage {
     if (this.options.content === null) {
       content = '';
     } else if (typeof this.options.content !== 'undefined') {
-      content = Util.resolveString(this.options.content);
+      content = Util.verifyString(this.options.content, RangeError, 'MESSAGE_CONTENT_TYPE');
```

Comment:
This currently allows empty strings, contrary to what the error states, did you miss the `false` for the `allowEmpty` parameter?